### PR TITLE
[CUBRIDMAN-128] Syntax modification of CREATE SERVER syntax

### PR DESCRIPTION
**Pull request by ctshim**
http://jira.cubrid.org/browse/CUBRIDMAN-128

* create server dbname support string too.

### CODE REVIEW

File: ko/sql/schema/server_stmt.rst [hunk 1]
```diff
@@ -47,6 +47,16 @@ CREATE SERVER
     *   *password_string*: 원격 접속할 데이터베이스에 접속할 때 사용하는 *user_name*\에 대한 패스워드 문자열.
     *   *properties_string*: 원격 접속할 데이터베이스에 접속할 때 사용하는 property 정보 문자열.(최대 2047바이트)	
     *   *server_comment_string*: 서버 정보에 대한 커멘트를 지정한다.(최대 1023바이트)
+
+.. note::
+
+    db_name과 user_name은식별자 형식과 문자열 리터럴 형식으로 모두 기술 할 수 있다.
```
Comment (hyunikn) [inline ko/sql/schema/server_stmt.rst]: space after '은'

File: en/sql/schema/server_stmt.rst [hunk 2]
```diff
@@ -47,6 +47,16 @@ For how to use the prepared server, refer to :ref:`dblink-clause`.
     *   *password_string*: Password string for *user_name* used to connect to the database to be accessed remotely.
     *   *properties_string*: Property information string used when connecting to the database to be accessed remotely. (up to 2047 bytes)	
     *   *server_comment_string*: Specifies comments about server information. (up to 1023 bytes)
+
+.. note::
+
+    db_name and user_name can be described in both identifier form and string literal form.
```
Comment (hyunikn) [inline en/sql/schema/server_stmt.rst]: 'given' seems to be more suitable than 'described'.

File: ko/sql/schema/server_stmt.rst (hunk 1 above)
Comment (mhoh3963) [inline ko/sql/schema/server_stmt.rst]:
```suggestion
    db_name과 user_name은 식별자 형식 또는 문자열 리터럴 형식으로 기술할 수 있다.

```

File: en/sql/schema/server_stmt.rst (hunk 2 above)
Comment (mhoh3963) [inline en/sql/schema/server_stmt.rst]: How about changing as following ?

db_name and user_name can be written in the form of identifier or string literal.

File: ko/sql/schema/server_stmt.rst (hunk 1 above)
Comment (ctshim) [reply ko/sql/schema/server_stmt.rst]: Thank you.

Comment (ctshim) [reply ko/sql/schema/server_stmt.rst]: Thank you

File: en/sql/schema/server_stmt.rst (hunk 2 above)
Comment (ctshim) [reply en/sql/schema/server_stmt.rst]: Thank you.

Comment (ctshim) [reply en/sql/schema/server_stmt.rst]: OK, thank you.

File: en/sql/schema/server_stmt.rst [hunk 3]
```diff
@@ -47,6 +47,17 @@ For how to use the prepared server, refer to :ref:`dblink-clause`.
     *   *password_string*: Password string for *user_name* used to connect to the database to be accessed remotely.
     *   *properties_string*: Property information string used when connecting to the database to be accessed remotely. (up to 2047 bytes)	
     *   *server_comment_string*: Specifies comments about server information. (up to 1023 bytes)
+
+.. note::
+
+    db_name and user_name can be written in the form of identifier or string literal.
+
+    
+    * Example identifier format
```
Comment (beyondykk9) [inline en/sql/schema/server_stmt.rst]:
```suggestion
    * Example of identifier format
```

**Review by beyondykk9 — APPROVED** [review]

File: en/sql/schema/server_stmt.rst (hunk 3 above)
Comment (ctshim) [reply en/sql/schema/server_stmt.rst]: Thank you, I will fix it.

**Review by zionyun — APPROVED** [review]

**Review by mhoh3963 — APPROVED** [review]

**Review by sungc-cubrid — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

**Review by hyunikn — APPROVED** [review]

**Pull request merged**